Fix nightly 2025-10-29

**What happened:**

https://github.com/gardenlinux/gardenlinux/actions/runs/18898643026/job/53943419542

```
 test_prefix = "gh-actions"
workspace = "test-18898643026-2401-azure-gardener_prod-amd64-933683cf"
features_image_name: gardener_prod
INFO: Flavor: azure-gardener_prod-amd64
INFO: Platform: azure
INFO: Provisioner: tofu
INFO: Features: ['log', 'sap', 'ssh', '_fwcfg', '_legacy', '_nopkg', '_prod', '_slim', 'base', 'server', 'cloud', 'azure', 'multipath', 'iscsi', 'nvme', 'gardener']
INFO: Architecture: amd64
INFO: Workspace: azure-gardener_prod-amd64-933683cf
ERROR: OpenTofu command failed: 
Workspace "azure-gardener_prod-amd64-933683cf" doesn't exist.

You can create this workspace with the "new" subcommand 
or include the "-or-create" flag with the "select" subcommand.

ERROR: OpenTofu error: Command '['tofu', 'workspace', 'select', 'azure-gardener_prod-amd64-933683cf']' returned non-zero exit status 1.
make: *** [Makefile:111: azure-gardener_prod-amd64-tofu-apply] Error 1
make: Leaving directory '/home/runner/work/gardenlinux/gardenlinux/tests/platformSetup'
Error: Process completed with exit code 2.
```

--- a/tests/platformSetup/Makefile
+++ b/tests/platformSetup/Makefile
@@ -73,7 +73,7 @@ CRE_CMD_privileged = --privileged
 # privileged is required for virt-copy-in/guestfish
 CRE_CMD_kvm = $(CRE_CMD_BASE) $(CRE_CMD_privileged) $(KVM_DEVICE)
 CRE_CMD_kvm_image = $(GL_REGISTRY)/gardenlinux/platform-test-kvm:$(GL_VERSION)
-CRE_CMD_tofu = $(CRE_CMD_BASE) -e TF_* -v ~/.aliyun:/root/.aliyun -e ALIBABA_* -v ~/.aws:/root/.aws -e AWS_* -v ~/.azure:/root/.azure -e azure_* -e ARM_* -e ACTIONS_* -v ~/.config/gcloud:/root/.config/gcloud -e GOOGLE_* -e CLOUDSDK_* -e OS_* --rm -v ~/.ssh:/root/.ssh:ro $(GL_REGISTRY)/gardenlinux/platform-test-tofu:$(GL_VERSION)
+CRE_CMD_tofu = $(CRE_CMD_BASE) -e TF_* -v ~/.aliyun:/root/.aliyun -e ALIBABA_* -v ~/.aws:/root/.aws -e AWS_* -v ~/.azure:/root/.azure -e azure_* -e ARM_* -e ACTIONS_* -e GITHUB_* -v ~/.config/gcloud:/root/.config/gcloud -e GOOGLE_* -e CLOUDSDK_* -e OS_* --rm -v ~/.ssh:/root/.ssh:ro $(GL_REGISTRY)/gardenlinux/platform-test-tofu:$(GL_VERSION)
 MKDIRS = ~/.aliyun ~/.aws ~/.azure ~/.config/gcloud
 
 .PHONY: all

--- a/tests/platformSetup/platformSetup.py
+++ b/tests/platformSetup/platformSetup.py
@@ -304,7 +304,7 @@ class Tofu:
             if github_run_id and github_run_number:
                 workspace = f"test-{github_run_id}-{github_run_number}-{flavor}-{self.paths.seed}"
             else:
-                workspace = f"{flavor}-{self.paths.seed}"
+                workspace = f"test-{flavor}-{self.paths.seed}"
 
             logger.info(f"Workspace: {workspace}")
             workspace_cmd = ["tofu", "workspace", "select", workspace]

